platform-hub-api/app/models/kubernetes_token.rb

```diff
@@ -64,6 +64,11 @@ def decrypted_token
     ENCRYPTOR.decrypt(token)
   end
 
+  def obfuscated_token
+    val = decrypted_token
+    val[0..30].gsub(/\w/, 'X') + val[31..35]
```

Comment:
Slightly faster (possibly):

```ruby
('X' * 30) + val[31..35]
```

Reply:
This one gets rid of dashes so leaving as is :)